apps/main-landing/src/components/top-bar/components/mobile/mobile.tsx

```diff
@@ -17,7 +18,7 @@ import { Socials } from './socials';
 export const Mobile = () => {
   return (
     <Dialog
-      className="fixed inset-x-4 bottom-3 top-[76px] flex flex-col rounded-[36px] border border-mint-400 bg-white/50 px-4 py-6 text-neutralGreen-900 backdrop-blur-[90px]" // top-[76px] is 64px height of navbar + 12px spacing, ideally it should be ref attached to nav to read component height in case it changes in future
+      className="fixed inset-x-4 bottom-3 top-[76px] flex flex-col rounded-[36px] bg-white/50 px-4 py-6 text-neutralGreen-900 backdrop-blur-[45px]" // top-[76px] is 64px height of navbar + 12px spacing, ideally it should be ref attached to nav to read component height in case it changes in future
```

Comment:
this backdrop blur should be also half of figma value? Damn :D

Reply:
It is 45px on figma, maybe it has changed :D 
![image](https://github.com/user-attachments/assets/4173f0a2-9ece-411d-a9f5-413c3af94037)